packages/flutter/lib/src/widgets/router.dart

```diff
@@ -64,31 +67,29 @@ class RouteInformation {
 /// button), parses route information into data of type `T`, and then converts
 /// that data into [Page] objects that it passes to a [Navigator].
 ///
-/// Additionally, every single part of that previous sentence can be overridden
-/// and configured as desired.
+/// Each part of this process can be overridden and configured as desired.
```

Comment:
Yes, this is much more professional, but I did really  love the old version. It looked like it was written by a really excited dev after a late night of hard work and a lot of coffee.

Reply:
I suspect it was originally written by me when I was designing the original Router API. :-)